modules/persistent-storage-cinder-maximum-volume.adoc

```diff
@@ -0,0 +1,7 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent_storage-cinder.adoc
+:_content-type: CONCEPT
+[id="persistent-storage-cinder-maximum-volume_{context}"]
+= Maximum number of Cinder volumes on a node
+By default, {product-title} supports a maximum of 256 Cinder volumes attached to one node, and the Cinder predicate that limits attachable volumes is disabled. To enable the predicate, add `MaxCinderVolumeCount` string to the `predicates` field in the scheduler policy.
```

Comment:
Looks good. But please note, in 4.11 and onwards, volume limits are driven by CSI predicates and that is enabled by default - https://kubernetes.io/docs/concepts/storage/storage-limits/